client/components/checklist/checklist.js

```diff
@@ -80,15 +83,25 @@ export default class Checklist extends PureComponent {
 			return { expandedTaskIndex: newExpandedTaskIndex }; // Expand
 		} );
 
-	toggleCompleted = () =>
-		this.setState( ( { hideCompleted } ) => ( { hideCompleted: ! hideCompleted } ) );
+	renderChecklistHeader = () => {
+		return (
+			<Card compact className="checklist__header">
+				<h2 className="checklist__header-progress-text">
```

Comment:
Could there be more accurately descriptive class name here? This seems like a leftover from the previous iteration.

Reply:
aah yes, that is left over. I'll update it.